crates/rolldown/src/runtime/runtime-tail-node.js

```diff
@@ -1,4 +1,4 @@
 // This is created by the rolldown. It's used to polyfill the global `require` function when you bundling code
 // with target `format: 'esm'` and `platform: 'node'`. It's used to make sure the global `require` could 
 // works in Node.js esm environment.
-export var __require = /* @__PURE__ */ createRequire(import.meta.url);
\ No newline at end of file
+export var require = /* @__PURE__ */ createRequire(import.meta.url);
```

Comment:
Keep using `__require` would be better. `require` seems to be easily conflict with others. If happed, it still breaks the worker case.

Maybe keep using `require` and allow to turn off `polyfill` for require.

@sapphi-red cc